src/main/scala/moonads/composers/package.scala

```diff
@@ -15,9 +15,9 @@ package object composers {
 
     implicit def IdT(implicit b: Monad[Id]): ComposableMonad[Id] =
 	builder[Id, IdT]
-	
-//    implicit def EitherT(implicit b: Monad[Either]): ComposableMonad[Either] =
-//	builder[Either, EitherT]
+
+    implicit def EitherT[L](implicit b: Monad[Either[L, ?]]): ComposableMonad[Either[L,?]] =
+  builder[Either[L,?], EitherT[?[_],L,?]]
```

Comment:
if you don't mind, I'm trying to keep the code as readable as possible, so, if you could align this line as with the other implementations, that would be really great